components/Payment/PaymentList/PaymentList.scss

```diff
@@ -0,0 +1,66 @@
+.payment-list {
+  display: flex;
+  flex-flow: column;
+  row-gap: 48px;
+
+  > *:not(:last-child) {
+    border-bottom: 1px solid rgba($color-white-alpha-300, 0.3);
+  }
```

Comment:
I think this is written in a complicated way.

I think it would be easier to understand if you add a boundary to the element class and remove it at the last element.